assets/src/components/activities/multi_input/MultiInputDelivery.tsx

```diff
@@ -111,12 +121,10 @@ export const MultiInputComponent: React.FC = () => {
             ? {
                 id: input.id,
                 inputType: input.inputType,
-                options: (uiState.model as MultiInputSchema).choices
-                  .filter((c) => input.choiceIds.includes(c.id))
-                  .map((choice) => ({
-                    value: choice.id,
-                    displayValue: toSimpleText(choice.content),
-                  })),
+                options: input.choiceIds.map((choiceId) => ({
```

Comment:
Use `choiceIds` ordering to determine the order to render. This order is determined by the Transformation (or lack thereof)